js/Docs/Bootstrap.Popup/Delegator.BS.ShowPopup.md

```diff
@@ -20,18 +20,22 @@ provides: [Delegator.BS.ShowPopup]
 
 (function(){
 
+  var triggers = {};
 
+  ['show', 'hide'].each(function(action){
 
-	Delegator.register('click', 'BS.showPopup', {
+    triggers['BS.'+ action + 'Popup'] = {
+      handler: function(event, link, api){
+        var target = api.get('target') ? link.getElement(api.get('target')) : document.id(link.get('href').split("#")[1]);
+        event.preventDefault();
+        if (!target) api.fail('Could not find target element to activate: ' + (api.get('target') || link.get('href')));
+        api.getBehavior().apply(target);
+        target.getBehaviorResult('BS.Popup')[action]();
+      }
+    };
 
-		handler: function(event, link, api){
-			var target = api.get('target') ? link.getElement(api.get('target')) : document.id(link.get('href').split("#")[1]);
-			event.preventDefault();
-			if (!target) api.fail('Could not find target element to activate: ' + (api.get('target') || link.get('href')));
-			api.getBehavior().apply(target);
-			target.getBehaviorResult('BS.Popup').show();
-		}
+  });
 
-	});
+  Delegator.register('click', triggers);
 
-})();
\ No newline at end of file
+})();
```

Comment:
This whole file is an artifact of sorts. When I created the sandbox, I pulled in all the docs from stuff that had it. I put placeholders for the ones that didn't. The way I did that was copied all the js files from the source directory and changed their file type to md, that way each file had a matching doc file, even though the contents were js. I was going to then go and replace all the content with API docs but never got around to it. TLDR, when making changes, feel free to ignore the /Docs/* files that have no markdown in them...